tests/components/top_bar.test.ts

```diff
@@ -532,6 +537,23 @@ describe("TopBar component", () => {
       app.destroy();
     }
   );
+
+  test("can insert an image", async () => {
+    fixture = makeTestFixture();
+    const fileUploader = new FileUploader();
+    let model = new Model();
+    let mountedSpreadsgeet = await mountSpreadsheet(fixture, { model, fileUploader });
+    model = mountedSpreadsgeet.parent.model;
+    await nextTick();
+    const sheetId = model.getters.getActiveSheetId();
+    const originalLenght = getFigureIds(model, sheetId).length;
+    await simulateClick(".o-topbar-menu[data-id='insert']");
+    await simulateClick(".o-menu-item[data-name='insert_image']");
+    const newLenght = getFigureIds(model, sheetId).length;
+    expect(newLenght).toEqual(originalLenght + 1);
```

Comment:
You can hardcode `1`, it's juste much more readable. For tests, readability matters a lot
```suggestion
    expect(getFigureIds(model, sheetId)).toHaveLength(1);
```